### PR TITLE
issue: change naming

### PR DESCRIPTION
**Pull request by dawidk-kainos**

### CODE REVIEW

File: src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html [hunk 1]
```diff
@@ -63,7 +63,7 @@ <h4 id="description">Click on slots to change availability</h4>
             </div>
             <hr>
             <div>
-                <button class="btn btn-primary uppercase" id="back-to-interview-btn" type="button" ng-show="isRecruiter" ng-click="goBackToRecruiterView()">back to recruiter view</button>
+                <button class="btn btn-primary uppercase" id="back-to-interview-btn" type="button" ng-show="isRecruiter" ng-click="goBackToRecruiterView()">back to availibilities</button>
```
Comment (scyganska) [inline src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html:66]: Should be availabilities

Comment (dawidk-kainos) [reply src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html:66]: it's to be deleted anyway

File: src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-manage-users.html [hunk 2]
```diff
@@ -20,5 +20,5 @@ <h3 class="center" id="manage-users-header">Manage users</h3>
                 </a>
             </tr>
         </table>
-        <button ui-sref="tdpr.recruiter.home" class ="btn btn-primary uppercase">back to recruiter view</button>
+        <button ui-sref="tdpr.recruiter.home" class ="btn btn-primary uppercase">back to availibilities</button>
```
Comment (scyganska) [inline src/main/resources/assets/html/partials/recruiter/tdpr-recruiter-manage-users.html:23]: Should be availabilities

File: src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html (hunk 1 above)
Comment (scyganska) [reply src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html:66]: What's that request for then?

Comment (dawidk-kainos) [reply src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html:66]: Other stuff. This I did just by the way.

**Pull request merged**